### ISSUE TITLE
Audio settings documentation

### ISSUE THREAD

**Issue by rcarmo** [neutrinolabs/xrdp#1521]
I've been looking through the Wiki, issue list and even parts of the source code, but I cannot find a good reference of how to tune audio quality (or force all clients to use a specific codec).

Does such a thing exist? Thanks!

**Comment by metalefty** [comment]
Regarding the document, it is not well maintained. Always help wanted.
Different distro have different manner. I don't have enough time to cover all distributions.

Regarding tuning audio quality, that is not possible so far. Auto select only.

**Comment by rcarmo** [comment]
Any plans to set the priority for negotiation? I have clients that support AAC getting uncompressed audio and clogging the network...

**Comment by metalefty** [comment]
If both the server and clients support AAC, AAC should be selected. The priority of AAC is always higher than PCM audio. Is your xrdp server compiled with AAC?

**Comment by rcarmo** [comment]
Yes, most definitely: https://github.com/rcarmo/ubuntu-xrdp/blob/master/xrdp/debian/rules#L54

**Comment by metalefty** [comment]
Indeed. Please also check `xrdp --version`.
